app/presenters/shopify/product_presenter.rb

```diff
@@ -0,0 +1,17 @@
+module Shopify
+  class ProductPresenter < Delegator
+    def description
+      html_renderer.render(model.description)
+    end
+
+    def vendor
+      'Default Vendor'
+    end
+
+    private
+
+    def html_renderer
+      Shopify::RedcarpetHTMLRenderer.new
```

Comment:
This is what I the most familiar with. I used Kramdown once and didn't quite like it. I have never researched about Rdiscout.